src/sspilib/_sec_context.py

```diff
@@ -340,18 +340,18 @@ class ClientSecurityContext(SecurityContext):
     context has been negotiated and is complete.
 
     Args:
-        target_name: The target service's name, typically the SPN.
-        flags: Custom ISC REQ flags to use.
         credential: A credential to use for authentication, defaults to the
             current user context if not set.
```

Comment:
There is no defaulting anymore.

Reply:
Thanks for the pickup, has been updated.